gcp_variant_transforms/libs/bigquery_util.py

```diff
@@ -420,6 +425,7 @@ def start_loading(self):
       self._start_one_load_job(self._remaining_load_jobs.pop())
 
     self._monitor_load_jobs()
+    return self._not_empty_suffixes
```

Comment:
So now we return value but don't use it in either bq_util_test or vcf_to_bq... Artifact of testing or is it going to be used somewhere in the future?

Reply:
We will use this returned value in #573 : only generate sample lookup optimized tables for not empty ones.